cmd/influxd/run/server.go

```diff
@@ -513,6 +513,8 @@ func (s *Server) Open() error {
 		return fmt.Errorf("open points writer: %s", err)
 	}
 
+	s.Monitor.WithLimiter(s.TSDBStore.EngineOptions.CompactionThroughputLimiter)
```

Comment:
I think I'm going to change this to `WithCompactThroughputLimiter` or something. Thinking about if in the future we want to add other limiters to the metrics collection for /debug/vars. :thinking: